resources/views/marketing/auth.blade.php

```diff
@@ -6,10 +6,10 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <meta name="csrf-token" content="{{ csrf_token() }}"/>
     <title>@yield('title', trans('app.application_title'))</title>
-    <link rel="manifest" href="/manifest.webmanifest">
+    <link rel="manifest" href="href="{{ asset('/manifest.webmanifest') }}"">
```

Comment:
I think it should be
```suggestion
    <link rel="manifest" href="{{ asset('/manifest.webmanifest') }}">
```